### DISCUSSION TITLE
the error gif...

### DISCUSSION THREAD

**Discussion by wongsimon** [dlemstra/Magick.NET#1284] [Help]
the error gif...

using (var images = new MagickImageCollection())
{
	pngPaths.ForEach(x =>
    {
        MagickImage item = new MagickImage(x);
        item.AnimationDelay = 4;
        images.Add(item);
    });
    images.Write(this.MapPath("eat2.gif", true), MagickFormat.Gif);
} 

right:
![32](https://user-images.githubusercontent.com/6326997/196885725-99f7122f-2ba5-4566-aca0-e6339ea4d433.gif)
error:
![eat2](https://user-images.githubusercontent.com/6326997/196762545-e6d5cdd5-fa76-464d-b9d7-4c06a847f3ce.gif)

how to hidden the other images when the last image is show.

**Comment by dlemstra** [comment]
You will need to set the `GifDisposeMethod` of the image.
